lib/reverse_string.rb

```diff
@@ -1,5 +1,21 @@
 def reverse_characters(str)
+#Ok this is a little better but im still not sure how to use the .join to get it back to a string
+#The array can be called back backwards but its not "saved" anywhere in that order so it cant be converted to a string
```

Comment:
Instead of using `puts` (which you don't need to use in this method at all), consider creating an empty array at the beginning of the method–and then push each character (starting from the end, of course) into that array. That way, you'll be saving the characters in their reverse order as you go through it.